### PR TITLE
[main] Bump Quarkus version to 2.16.8.Final

### PR DESCRIPTION
**Pull request by kie-ci3**
Please review and merge
- https://github.com/kiegroup/kogito-pipelines/pull/1014
- https://github.com/kiegroup/kogito-docs/pull/386
- https://github.com/kiegroup/kogito-images/pull/1623
- https://github.com/kiegroup/kogito-runtimes/pull/3117
- https://github.com/kiegroup/kogito-examples/pull/1739

### CODE REVIEW

File: kogito-build/kogito-dependencies-bom/pom.xml [hunk 1]
```diff
@@ -16,10 +16,10 @@
 
   <properties>
     <!-- this version property is used in plugins but also in dependencies too -->
-    <version.io.quarkus>2.16.7.Final</version.io.quarkus>
+    <version.io.quarkus>2.16.8.Final</version.io.quarkus>
     <version.io.quarkus.quarkus-test>${version.io.quarkus}</version.io.quarkus.quarkus-test>
     <version.org.springframework.boot>2.6.6</version.org.springframework.boot>
-    <version.org.apache.kafka>3.4.0</version.org.apache.kafka>
+    <version.org.apache.kafka>3.3.2</version.org.apache.kafka>
```
Comment (radtriste) [inline kogito-build/kogito-dependencies-bom/pom.xml:22]: @cristianonicolai this is a regression ... was kafka updated manually recently ?

File: kogito-build/kogito-dependencies-bom/pom.xml [hunk 2]
```diff
@@ -122,7 +122,7 @@
     <!-- see: https://maven.apache.org/surefire/maven-surefire-plugin/examples/fork-options-and-parallel-execution.html#parallel-test-execution-and-single-thread-execution -->
     <version.com.github.stephenc.jcip>1.0-1</version.com.github.stephenc.jcip>
     <version.black.ninia>4.1.1</version.black.ninia>
-    <version.com.google.guava>32.0.1-jre</version.com.google.guava>
+    <version.com.google.guava>31.1-jre</version.com.google.guava>
```
Comment (cristianonicolai) [inline kogito-build/kogito-dependencies-bom/pom.xml:125]:
```suggestion
    <version.com.google.guava>32.0.1-jre</version.com.google.guava>
```

File: kogito-build/kogito-dependencies-bom/pom.xml (hunk 1 above)
Comment (cristianonicolai) [inline kogito-build/kogito-dependencies-bom/pom.xml:22]:
```suggestion
    <version.org.apache.kafka>3.4.0</version.org.apache.kafka>
```

Comment (cristianonicolai) [reply kogito-build/kogito-dependencies-bom/pom.xml:22]: CVE related fix, will skip downgrade

**Review by cristianonicolai — APPROVED** [review]

**Comment by Ginxo** [comment]
Jenkins run kogito-runtimes tests

**Pull request merged**